src/gallery.js

```diff
@@ -1,7 +1,64 @@
 let gallery = [],
 	current = 0
 
-const hideFullscreenGallery = ( galleryContainer ) => {
+const getLoadingContainer = ( doc ) => {
+		const component = doc.createElement( 'div' ),
+			spinnerContainer = doc.createElement( 'div' ),
+			spinner = doc.createElement( 'div' ),
+			bounceBegin = doc.createElement( 'div' ),
+			bounceEnd = doc.createElement( 'div' ),
+			stillLoading = doc.createElement( 'div' )
+
+		component.classList.add( 'wp-gallery-popup-loading' )
+		spinnerContainer.classList.add( 'wp-gallery-popup-loading-spinner-container' )
+		spinner.classList.add( 'wp-gallery-popup-loading-spinner' )
+		bounceBegin.classList.add( 'wp-gallery-popup-loading-bounce' )
+		bounceEnd.classList.add( 'wp-gallery-popup-loading-bounce' )
+		stillLoading.classList.add( 'wp-gallery-popup-loading-still' )
+		stillLoading.innerHTML = 'Still loading'
+
+		spinner.appendChild( bounceBegin )
+		spinner.appendChild( bounceEnd )
```

Comment:
Does anyone else also get a linter error with this?

```
[ bounceBegin, bounceEnd ].forEach( bounce => {
   spinner.appendChild( bounce )
} )
```

```
ERROR in ./src/gallery.js
Module Error (from ./node_modules/eslint-loader/dist/cjs.js):


/Users/emedina/internet/wikimedia/wikipedia-previews/src/gallery.js
  17:4   error  Unexpected newline between object and [ of property access  no-unexpected-multiline
  17:17  error  Unexpected use of comma operator
```

https://eslint.org/docs/2.0.0/rules/no-unexpected-multiline

Reply:
```diff
- [ bounceBegin, bounceEnd ].forEach( bounce => {
+ const bounceArray = [ bounceBegin, boundEnd ]
+ bounceArray.forEach( bounce => {
   spinner.appendChild( bounce )
} )
```

can fix that